modules/gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-verifying-argocd-agent-installation-in-managed-or-autonomous-mode_{context}"]
+= Verifying the Argo CD Agent installation in Managed Mode or Autonomous Mode
+
+Use this procedure to verify that the Argo CD Agent is correctly installed and connected to the Principal cluster in managed or autonomous mode.
+
+.Procedure
+
+. Create an `AppProject` in the agent namespace.
```

Comment:
@jgwest this is for autonomous mode only, right?

Reply:
(Thanks @varshab1210! I thought I had left a comment on this: the comment DID exist, but it had not been submitted)

This step, creating an AppProject, is a *setup* step for autonomous mode, not a *verification* step.

This step can thus be moved to autonomous mode install section, as in the gist: https://gist.github.com/jgwest/da80a952732ca026e21c156cca6ab5fe (step 7)